content/docs/cms.md

```diff
@@ -92,12 +92,33 @@ export default function SomeComponent() {
 }
 ```
 
+## Disabling / Enabling the CMS
+
+The CMS can be enabled or disabled via methods or configuration on the CMS object.
+
+```js
+import * as React from 'react'
+import { useCMS } from 'tinacms'
+
+// <ExitButton /> is assumed to be nested inside of a <Tina> component
+export default function ExitButton() {
+  const cms = useCMS()
+
+  return (
+    <button onClick={cms.enabled ? cms.disable() : cms.enable()}>
```

Comment:
Every time I look at this I think that maybe there should just be a `cms.toggle()` function as well

Reply:
yessss

Reply:
https://github.com/tinacms/tinacms/pull/1265